### PR TITLE
SlideShow_skeleton cooking page

### PR DESCRIPTION
**Pull request by thuyvy119**
Done issue #8: A slider with static slideshow that displays cooking steps including images and descriptions for each step in the cooking page:

- ChakraCarousel component with Slider (overall layout and positioning of the carousel), Track (render items in horizontal layout) and Items components: 
  - Slider provides incrementing and decrementing buttons, use useBoundingRect hook to calculate the slider width.
  - Track handles drag and swipe gestures, uses framer motion library for animating the item's positions.
  - Item handles focus and blur events to update the active item state.

https://github.com/hungngodev/Flavorie/assets/45311428/ec36e210-024a-4798-8ac5-cbe03a87176e

### CODE REVIEW

File: client/src/App.tsx [hunk 1]
```diff
@@ -15,18 +18,18 @@ const theme = extendBaseTheme({
 const router = createBrowserRouter([
   {
     path: "/",
-    element: <div> Your components </div>,
+    element: <ImageSlide />,
```
Comment (hungngodev) [inline client/src/App.tsx]: Change it to the original so we can merge it

**Review by hungngodev — CHANGES_REQUESTED** [review]
Change a little bit at App.tsx

**Review by hungngodev — COMMENTED** [review]
Everything is perfect!! Great jobbb ten thumbs up! However, I just want verify if it is working correctly so can you hop on Discord so we can discuss it? You can go on to the next task!

**Review by hungngodev — CHANGES_REQUESTED** [review]
A video of demo pleasee!

**Comment by thuyvy119** [comment]
> A video of demo pleasee!

Done nhe.

**Review by hungngodev — CHANGES_REQUESTED** [review]
Can you fix again the merge conflict?

**Review by hungngodev — APPROVED** [review]
Done

**Review by hungngodev — APPROVED** [review]
Doneee

**Pull request merged**